### DISCUSSION TITLE
Welcome to tasty-discussion Discussions!

### DISCUSSION THREAD

**Discussion by PermeationLoop** [Yamabuki-bakery/tasty-discussion#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SaltedFishesNG** [comment]
话说这合理吗，数字怎么用的是emoji字体啊（
虽然我不知道怎么用chromium系的开发者工具查看字体，但这个问题可不是firefox的锅，也不是linux的锅。
<img width="413" height="286" alt="2026-02-10 18-38-48" src="https://github.com/user-attachments/assets/799ca61a-60c6-47c5-86ec-c9811051d091" />

引发这个问题的代码在
https://ex-tasty.com/comments-css/dark_high_contrast.css
只要将第72行的`ui-monospace`删掉就没问题了

**Reply by PermeationLoop** [reply]
立刻嚴肅修復！😅 但是我手頭上沒有 Linux 電腦，只能用 Windows 上的 Firefox 湊合一下我先逝逝

**Reply by PermeationLoop** [reply]
我沒有 Linux，我用安卓手機修的，現在 works on my phone 了，妳再看看正常了沒（

**Reply by SaltedFishesNG** [reply]
的确没问题了qwq
~~其实我昨天晚上认为不是firefox的问题，是因为我看linux chromium和androidfirefox也有这问题（~~
~~然后早上一看android的chrome没法复现~~
